src/screens/AdminPortal/OrgList/modal/OrganizationModal.tsx

```
@@ -91,208 +89,199 @@ const OrganizationModal: React.FC<InterfaceOrganizationModalProps> = ({
+        placeholder={tCommon('description')}
+        value={formState.description}
+        onChange={(val) => {
+          if (val.length <= 200) {
+            setFormState({ ...formState, description: val });
+          }
+        }}
+        required
+        data-testid="modalOrganizationDescription"
+        autoComplete="off"
+      />
+      <label className="form-label">{tCommon('address')}</label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)